recroot/src/App.js

```diff
@@ -6,18 +6,60 @@ class App extends Component {
 
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to Recroot! (powered by React)</h1>
-        </header>
-        <p className="App-intro">
-          Recroot is an app allowing you to connect with your fellow game convention attendees to quickly get a game session started.  
-        </p>
-        <button onClick={() => alert("Alert button clicked")}>Click for Alert</button>
+      <div>
+        <div className="App">
+          <header className="App-header">
+            <img src={logo} className="App-logo" alt="logo" />
+            <h1 className="App-title">Welcome to Recroot! (powered by React)</h1>
+          </header>
+          <p className="App-intro">
+            Recroot is an app allowing you to connect with your fellow game convention attendees to quickly get a game session started.  
+          </p>
+          <button onClick={() => alert("Alert button clicked")}>Click for Alert</button>
+        </div>
+        <div className="EventForm">
+          <EventForm />
+        </div>
```

Comment:
Is App.js a controller? Does that mean, at some point, this gets moved into a different view file?